packages/core/src/builder.class.ts

```diff
@@ -2549,7 +2541,7 @@ export class Builder {
       for (const key of properties) {
         const value = options[key];
         if (value !== undefined) {
-          if (apiEndpoint === 'query') {
+          if (format === 'solid' || format === 'react') {
```

Comment:
if we move `isApiCallForCodegen` above this line we can reuse it here
```suggestion
          if (isApiCallForCodegen) {
```